db/schema.rb

```diff
@@ -25,6 +25,15 @@
     t.datetime "updated_at"
   end
 
+  create_table "comments", force: :cascade do |t|
```

Comment:
Could we remove these from the schema on this branch? This is not relevant to this branch.